README.md

```diff
@@ -30,6 +30,14 @@ log-20160702.txt
 
 > **Important:** By default, only one process may write to a log file at a given time. See _Shared log files_ below for information on multi-process sharing.
 
+### Date placeholders
```

Comment:
Will make the change. Only called it placeholder as a copy of the chapter above.